packages/coinjoin/src/client/CoinjoinRound.ts

```diff
@@ -225,12 +227,19 @@ export class CoinjoinRound extends EventEmitter {
                 );
 
                 this.addresses.forEach(addr =>
-                    prison.detain(addr.address, {
+                    prison.detain(addr.scriptPubKey, {
```

Comment:
just curious, why this change?

Reply:
output registration checks if [scriptPubKey is detained](https://github.com/trezor/trezor-suite/blob/develop/packages/coinjoin/src/client/round/outputRegistration.ts#L52)